modules/Components/src/client/PickList.coffee

```diff
@@ -544,15 +544,16 @@ class window.EditablePickListSelect2Controller extends EditablePickListSelectCon
 class window.ThingLabelComboBoxController extends Backbone.View
 
 	initialize: ->
-		console.dir @options
 		@thingType = @options.thingType
 		@thingKind = @options.thingKind
 		@labelType = if @options.labelType? then @options.labelType else null
 		@placeholder = if @options.placeholder? then @options.placeholder else null
 		@queryUrl = if @options.queryUrl? then @options.queryUrl else null
+		unless @queryUrl? or (@thingType? and @thingKind?)
+			alert("ThingLabelComboBoxController URL misconfigured - crash to follow")
 
 	render: =>
-		@$el.select2
+		@selectController = @$el.select2
```

Comment:
I think it's ok to leave it in there because you might need it for listening to event triggers. I was just wondering if there was an example where you used it for that.